README.md

```diff
@@ -205,20 +197,13 @@ in recently processed blocks to save to disk.
 ```
 
 ## Testing with rosetta-cli
-To validate `rosetta-bitcoin`, [install `rosetta-cli`](https://github.com/coinbase/rosetta-cli#install)
-and run one of the following commands:
-* `rosetta-cli check:data --configuration-file rosetta-cli-conf/testnet/config.json`
-* `rosetta-cli check:construction --configuration-file rosetta-cli-conf/testnet/config.json`
-* `rosetta-cli check:data --configuration-file rosetta-cli-conf/mainnet/config.json`
-
-## Future Work
-* Publish benchamrks for sync speed, storage usage, and load testing
-* [Rosetta API `/mempool/transaction`](https://www.rosetta-api.org/docs/MempoolApi.html#mempooltransaction) implementation
-* Add CI test using `rosetta-cli` to run on each PR (likely on a regtest network)
-* Add performance mode to use unlimited RAM (implementation currently optimized to use <= 16 GB of RAM)
-* Support Multi-Sig Sends
+To validate `rosetta-bitcoin`, [install `rosetta-cli`](https://github.com/coinbase/rosetta-cli#install) and run one of the following commands:
+* `rosetta-cli check:data --configuration-file rosetta-cli-conf/testnet/config.json` - This command validates that the Data API implementation is correct using the Bitcoin `testnet` node. It also ensures that the implementation does not miss any balance-changing operations.
+* `rosetta-cli check:construction --configuration-file rosetta-cli-conf/testnet/config.json` - This command validates the Construction API implementation. It also verifies transaction construction, signing, and submissions to the `testnet` network.
+* `rosetta-cli check:data --configuration-file rosetta-cli-conf/mainnet/config.json` - This command validates that the Data API implementation is correct using the Bitcoin `mainnet` node. It also ensures that the implementation does not miss any balance-changing operations.
 
-_Please reach out on our [community](https://community.rosetta-api.org) if you want to tackle anything on this list!_
+## Issues
+Interested in helping fix issues in this repository? You can find to-dos in the [Issues](https://github.com/coinbase/rosetta-bitcoin/issues) section with the `help wanted` tag. Be sure to reach out on our [community](https://community.rosetta-api.org) before you tackle anything on this list.
```

Comment:
we can remove the `help wanted` here too.

Reply:
Fixed in next commit.